webapp/src/Controller/Jury/ImportExportController.php

```diff
@@ -436,6 +436,13 @@ protected function getResultsHtml(Request $request, bool $useIcpcLayout): Respon
 
         $collator = new Collator('en_US');
         $collator->sort($honorable);
```

Comment:
What does this do if we afterwards sort it again?

Reply:
Do we? This is honorable and below is ranked.

Reply:
Ok, and rank is decided by time instead of name.